.husky/commit-msg

```diff
@@ -0,0 +1 @@
+hmpo-commitmsg $1 '(CFPF|DD|TRP|FOR)'
```

Comment:
Don't need to enforce this for open-source libs, changes may come from the community and this will mean nothing to them